sequencer/src/api.rs

```diff
@@ -295,17 +297,17 @@ impl<N: network::Type> Borrow<HotShotSequencingConsensusApi<SeqTypes, Node<N>>>
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct NamespaceProofQueryData {
-    proof: NamespaceProofType,
-    nmt_root: NMTRoot,
+    pub proof: NamespaceProofType,
+    pub header: Header,
```

Comment:
What's the reason for making these public if we also have getters?

Reply:
So that you can move out of the fields